blog/2021-08-11-airbyte-case-study.md

```diff
@@ -0,0 +1,88 @@
+---
+tags:
+  - Airbyte
+  - Data pipeline
+  - Machine learning
+  - microservice-orchestration
+  - case study
+posted_on_: 2021-08-11T23:06:09Z
+slug: airbyte-case-study
+title: 'Durable, scalable EL(T) with MIT OSS Airbyte and Temporal'
```

Comment:
really want to go with the "MIT OSS"? too many acronyms in a title?

Reply:
Changed